### PR TITLE
feat: add changelog tempate.

### LINKED ISSUE #112: Add changelog file

### PR DESCRIPTION
**Pull request by zhouaihui**
Gona fill the content if release is ready.

Will close #112 when new version is released.

### CODE REVIEW

**Review by NKcqx — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Types of changes
+`Added ` for new features.
+`Changed` for changes in existing functionality.
+`Deprecated` for soon-to-be removed features.
+`Removed` for now removed features.
+`Fixed` for any bug fixes.
+`Security` in case of vulnerabilities.
+
+## [0.1.0] - 2024-01-09
```
Comment (jovany-wang) [inline CHANGELOG.md:15]: I remember that we should start the initial version at 0.1.1 as secretflow is using 0.1.0 now.

Comment (zhouaihui) [reply CHANGELOG.md:15]: This is a template only to show how to fill the content.
And i'm gona complete the content when release is ready (docs is not ready).

Comment (jovany-wang) [reply CHANGELOG.md:15]: Get.

**Review by jovany-wang — APPROVED** [review]

**Pull request merged**